src/test/scala/org/apache/spark/deploy/k8s/integrationtest/backend/IntegrationTestBackend.scala

```diff
@@ -29,9 +29,13 @@ private[spark] trait IntegrationTestBackend {
 }
 
 private[spark] object IntegrationTestBackendFactory {
-  def getTestBackend(): IntegrationTestBackend = {
-    Option(System.getProperty("spark.kubernetes.test.master"))
-      .map(new GCETestBackend(_))
-      .getOrElse(MinikubeTestBackend)
+  def getTestBackend: IntegrationTestBackend = {
+    val deployMode = Option(System.getProperty("spark.kubernetes.test.deployMode"))
+      .getOrElse("minikube")
+    if (deployMode == "minikube") {
+      MinikubeTestBackend
+    } else {
+      new GCETestBackend()
```

Comment:
Since this is an `object` now, this should just be `GCETestBackend` (no `new` operator).

Reply:
You were probably looking at an old commit. I did that and actually renamed the class to `CloudTestBackend`.